# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by scripts/Create-CopyPatch.
# 
# T2 SDE: package/*/aboot/hotfix.patch
# Copyright (C) 2021 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- aboot.git/Makefile.vanilla	2000-01-01 00:00:00.000000000 +0100
+++ aboot.git/Makefile	2021-04-23 17:57:04.222524658 +0200
@@ -61,7 +61,7 @@
 ifeq ($(TESTING),)
 ABOOT_OBJS = \
 	head.o aboot.o cons.o utils.o \
-	zip/misc.o zip/unzip.o zip/inflate.o
+	zip/misc.o zip/unzip.o zip/inflate.o lib/isolib.o
 else
 ABOOT_OBJS = aboot.o zip/misc.o zip/unzip.o zip/inflate.o
 endif
--- aboot.git/Makefile.vanilla	2021-04-23 18:01:52.946523161 +0200
+++ aboot.git/Makefile	2021-04-23 18:02:03.416523107 +0200
@@ -96,7 +96,7 @@
 	make -C doc/man install-gz
 
 install: tools/abootconf tools/e2writeboot tools/isomarkboot \
-	sdisklabel/swriteboot install-man
+	sdisklabel/swriteboot # install-man
 	install -d $(bindir) $(bootdir)
 	install -c tools/abootconf $(bindir)
 	install -c tools/e2writeboot $(bindir)
